### PR TITLE
CFD-377: Group approval workflow : Add group management overview.

### PR DESCRIPTION
**Pull request by nader77**
#548 

selection_077

### CODE REVIEW

**Comment by nader77** [comment]
@zero2one @HelenaEksler Some questions:
* This table view should be accessed only by content administrator ?
* Should it have a pager ?, how many items?
* I have added a menu tab to get to this table view, Called "group" in the admin menu, Is it ok or should not have menu link ?

**Comment by nader77** [comment]
Update: Added behat tests.

File: capacity4more/behat/features/bootstrap/FeatureContext/Group.php [hunk 1]
```diff
@@ -164,6 +164,21 @@ public function iChangeAccessOfGroupTo($title, $access) {
...
   /**
+   * @Then /^I should see the group management table$/
+   */
+  public function iShouldSeeGroupTable() {
+    $steps = array();
+    $steps[] = new Step\When('I should see "Name"');
+    $steps[] = new Step\When('I should see "Architecture"');
+    $steps[] = new Step\When('I should see "Status"');
+    $steps[] = new Step\When('I should see "Owner"');
+    $steps[] = new Step\When('I should see "mariecurie"');
```
Comment (HelenaEksler) [inline capacity4more/behat/features/bootstrap/FeatureContext/Group.php]: Maybe it's better to add parameter for user name?

**Comment by zero2one** [comment]
@nader77 @HelenaEksler 

This table view should be accessed only by content administrator ? 
=> **YES** : all roles who have "administer content" permission.

Should it have a pager ?, how many items?
=> **YES** : full pager, 30 items /page.

I have added a menu tab to get to this table view, Called "group" in the admin menu, Is it ok or it should not have a menu link ?
=> That is fine for now, we will group platform specific admin pages later.

**Comment by nader77** [comment]
@HelenaEksler PR is ready for review, Test will fail because i checkout out from the status field branch (#537).

**Comment by HelenaEksler** [comment]
@nader77 Looks fine. Please, add screenshot with pager and assign to Peter.

**Comment by nader77** [comment]
Pager when there's more than 30 groups:
selection_080

**Comment by nader77** [comment]
@zero2one PR is ready.

**Comment by zero2one** [comment]
@nader : I forgot the exposed filters above the table view.

#### Filter

Add an **Exposed filter** above the view with following fields:

* **Name** : Search groups based on the group name (text field, field contains).
* **Status** : Filter the groups based on the status (select dropdown).

**Comment by nader77** [comment]
@zero2one Added the exposed filter to the groups table:

PR is ready for code review: 
groups-2015-03-25_17

**Pull request merged**